examples/plot_structure_basic/plot_inferred_structure.py

```diff
@@ -0,0 +1,223 @@
+"""
+Plot an inferred 3-d structure with matplotlib
+==============================================
+
+This example showcases how to plot an inferred 3-d structure output by
+PASTIS using matplotlib.
+"""
+
+import numpy as np
+import itertools
+from mpl_toolkits.mplot3d import axes3d
+import matplotlib
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy import interpolate
+
+##############################################################################
+# Get a dataframe for the inferred structure's coordinates.
+df = pd.read_csv('data/struct_inferred.000.coords', delimiter=" ", header=None)
```

Comment:
You might be able to read in the coordinates directly as a numpy array with np.loadtxt. I think it would work "out-of-the-box" without the need of changing any default options.

Reply:
Yup, this worked.